website/src/components/subscriptions/overview/SubscriptionDisplay.tsx

```diff
@@ -0,0 +1,203 @@
+import type { ReactNode } from 'react';
+import type {
+    CountTrigger,
+    EvaluationInterval,
+    LapisFilter,
+    ProportionTrigger,
+    Subscription,
+} from '../../../types/Subscription.ts';
+import { type DateWindow, dateWindowConfig } from '../../../types/DateWindow.ts';
+import type { NotificationChannels } from '../../../types/NotificationChannels.ts';
+
+export function SubscriptionContent({ children, className = '' }: { children: ReactNode; className?: string }) {
```

Comment:
```suggestion
export function SubscriptionContent({ children, className = '' }: PropsWithChildren<WithClassName>) {
```

Where `type WithClassName<T = {}> = T & { className?: string };`. We have the same props a couple times now (e.g. in `CardHeader`, we could make it a pattern)